frontend/src/main.jsx

```diff
@@ -13,14 +12,15 @@ import LogHistory from "./pages/LogHistory.jsx";
 import UploadPhotos from "./pages/UploadPhoto.jsx";
 import ManualEdit from "./pages/ManualEdit.jsx";
 import "./index.css";
+import LogCode from "./pages/LogCode.jsx";
 
 createRoot(document.getElementById("root")).render(
   <AuthProvider>
     <BrowserRouter>
       <Routes>
         <Route path="/login" element={<Login />} />
         <Route path="/homepage" element={<Home />} />
-        <Route path="/signup" element={<SignUp />} />
+        <Route path="/logcode" element={<LogCode />} />
```

Comment:
logCode should be in protected routes (the full flow for going from signin to accessing the logcode page will be implemented later, but the idea is that the logCode page will only be accessible for accounts that have been registered with email and password)

Could you also rename "homepage" to "home"? Currently if I try to access a protected route without logging in, it will direct the user back to home, and it will say page not found